### ISSUE TITLE
Bug in saving to File From Bitmap

### ISSUE THREAD

**Issue by TheOriginalBytePlayer** [skia4delphi/skia4delphi#73]
It is currently CASE Sensitive to the extension you give a bitmap, so entering .PNG causes it to say it doesn't know the format.

If you change the following function from using CompareText (which requires an = 0) to SameText (which does not) then it becomes case independent.

```
class function TSkEncodedImageFormatHelper.FromExtension(
  const AExtension: string): TSkEncodedImageFormat;
begin
  if (SameText('.jpg', AExtension)) or (SameText('.jpeg', AExtension)) then
    Result := TSkEncodedImageFormat.JPEG
  else if SameText('.webp', AExtension) then
    Result := TSkEncodedImageFormat.WEBP
  else if SameText('.png', AExtension) then
    Result := TSkEncodedImageFormat.PNG
  else
    raise ESkArgumentException.Create(SInvalidExtension);
end;

```

**Comment by viniciusfbb** [comment]
@TheOriginalBytePlayer Thank you very much to report the issue!

You are right, the image format extensions check is case sensitive in last release. I committed just now, a fix that will be available in next release.

**Comment by viniciusfbb** [comment]
Fixed in new version [v3.3.0](https://github.com/skia4delphi/skia4delphi/releases)
